x-pack/plugins/observability_solution/logs_shared/public/components/log_stream/log_stream.tsx

```diff
@@ -297,9 +298,9 @@ Read more at https://github.com/elastic/kibana/blob/main/src/plugins/kibana_reac
   );
 };
 
-const LogStreamContainer = euiStyled.div`
+const LogStreamContainer = styled.div`
   display: flex;
-  background-color: ${(props) => props.theme.eui.euiColorEmptyShade};
+  background-color: ${euiThemeVars.euiColorEmptyShade};
```

Comment:
> [!IMPORTANT]
> The usage of `euiThemeVars` is deprecated as per this [EUI meta issue](https://github.com/elastic/kibana/issues/199715#json-tokens).

In order to still make this work, the right approach to go for is changing this to:
```ts
const LogStreamContainer = styled.div`
  display: flex;
  background-color: ${props => props.theme.euiTheme.colors.emptyShade};
`;
```

This will pull the right color from the theme. Be aware that, to get the correct types and have TS not complaining, you also need to add on the root folder of this plugin a `emotion.d.ts` file:
```ts
import '@emotion/react';
import type { UseEuiTheme } from '@elastic/eui';

declare module '@emotion/react' {
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface Theme extends UseEuiTheme {}
}
```